dronegen/promote.go

```diff
@@ -43,6 +43,25 @@ func promoteBuildPipelines() []pipeline {
 	ociPipeline.Trigger.Target.Include = append(ociPipeline.Trigger.Target.Include, "promote-distroless")
 	promotePipelines = append(promotePipelines, ociPipeline)
 
+	tbotOCIPipeline := ghaBuildPipeline(ghaBuildType{
```

Comment:
Publishing (was "promotion") on master is now fronted by the `tag-publish.yaml` workflow in teleport.e and will no longer be driven by drone. This is currently being backported to the release branches. Could you add the call to your workflow there and not here in drone?